### ISSUE TITLE
Many transitions not work well

### ISSUE THREAD

**Issue by cuiyajie** [selemondev/nuxt-ui-vue#135]
For example, 
```
const notificationTransitions = {
  enter: variant.value.transitionEnterActiveClass,
  enterFrom: variant.value.transitionEnterFromClass,
  enterTo: variant.value.transitionEnterToClass,
  leave: variant.value.transitionLeaveActiveClass,
  leaveFrom: variant.value.transitionLeaveFromClass,
  leaveTo: variant.value.transitionLeaveToClass,
}
```

should be:
```
const notificationTransitions = {
  enterClass: variant.value.transitionEnterActiveClass,
  enterFromClass: variant.value.transitionEnterFromClass,
  enterToClass: variant.value.transitionEnterToClass,
  leaveClass: variant.value.transitionLeaveActiveClass,
  leaveFromClass: variant.value.transitionLeaveFromClass,
  leaveToClass: variant.value.transitionLeaveToClass,
}
```
you should check the component `Transition` is from '@headlessui/vue' or 'vue'

**Comment by selemondev** [comment]
Where don't these transitions work?

**Comment by cuiyajie** [comment]
Notification

**Comment by selemondev** [comment]
This issue has now been resolved with the latest release of Nuxt UI Vue v1.0.1-beta.3.
